src/test/java/com/cloudbees/plugins/credentials/cli/CLICommandsTest.java

```diff
@@ -306,6 +309,47 @@ public void deleteSmokes() throws Exception {
         assertThat(store.getDomainByName("smokes"), nullValue());
     }
 
+    @Test
+    public void listCredentialsAsXML() throws IOException {
+        Domain smokes = new Domain("smokes", "smoke test domain",
+                Collections.<DomainSpecification>singletonList(new HostnameSpecification("smokes.example.com", null)));
+        UsernamePasswordCredentialsImpl smokey =
+                new UsernamePasswordCredentialsImpl(CredentialsScope.GLOBAL, "smokes-id", "smoke testing", "smoke",
```

Comment:
can you add a credential at system scope to the test?

Reply:
Done.